wannierberri/__result.py

```diff
@@ -45,6 +45,11 @@ def __sub__(self,other):
 # writing to a file
     def write(self,name):
         raise NotImprementedError()
+        
+# saving as binary
+    def save(self,name):
+        raise NotImprementedError()
```

Comment:
Imple (same above)

Reply:
Fixed. Is that all? 

Any thoughts on the file format? We probably do not want to change it (and the reference files) often in the future. Is there any reason why it might be inconvenient?